lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -3615,7 +3615,9 @@ SPIRVValue *LLVMToSPIRVBase::transIntrinsicInst(IntrinsicInst *II,
     return nullptr;
   }
   case Intrinsic::bitreverse: {
-    BM->addCapability(CapabilityShader);
+    if (!BM->isAllowedToUseExtension(ExtensionID::SPV_KHR_bit_instructions)) {
+      BM->addCapability(CapabilityShader);
```

Comment:
@svenvh should we remove this handling after https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/2270 ?

Reply:
This seems to be the only (remaining) case where we emit `CapabilityShader` currently, so it makes sense to remove it indeed.  I'll open a PR shortly.

Reply:
https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/2287